### ISSUE TITLE
Server exited unexpectedly after selecting text with cursor

### ISSUE THREAD

**Issue by Solarunit** [tmux/tmux#3855]
### Issue description

I recently noticed that tmux crashes after selecting text with the cursor.
After selection I see _server exited unexpectedly_

But selection with holding `shift` still works

### Required information

* tmux version (`tmux -V`). tmux 3.3a
* Platform (`uname -sp`). Linux
* $TERM inside and outside of tmux (`echo $TERM`). alacritty, tmux-256color

**Comment by Solarunit** [comment]
I found the same closed issue https://github.com/microsoft/CBL-Mariner/issues/6598

**Comment by arouene** [comment]
Same problem, can reproduce with only 
```
set -g mouse on
```
as configuration

**Comment by Solarunit** [comment]
Yes, `set -g mouse on` should be enabled. I can reproduce this in other terminals, kitty, gnome-terminal.
I noticed this problem just recently.

I have tried **tmux 3.4** on another system, no problem there.
